### PR TITLE
Implemented function to calculate weekly training plan based on age 

### PR DESCRIPTION
**Pull request by bscheltinga**
Implementation of Issue #6  
Could you please check this Sven?

### CODE REVIEW

File: Rasa_Bot/actions/actions.py [hunk 1]
```diff
@@ -29,8 +30,9 @@ def name(self):
         return "action_get_plan_week"
 
     async def run(self, dispatcher, tracker, domain):
-        
-        plan = "Sure, you should do 2 half-hour running sessions. And please read through this psycho-education: www.link-to-psycho-education.nl."
+        age = 30 # We should get this value from a database.
+        WeeklyKilometers = weekly_kilometers(age) # Calculates weekly kilometers based on age
```
Comment (svenvanderburg) [inline Rasa_Bot/actions/actions.py:34]: Variable names in python are snake_case:
```suggestion
        weekly_kilometers = weekly_kilometers(age) # Calculates weekly kilometers based on age
```

File: Rasa_Bot/actions/actions.py [hunk 2]
```diff
@@ -29,8 +30,9 @@ def name(self):
         return "action_get_plan_week"
 
     async def run(self, dispatcher, tracker, domain):
-        
-        plan = "Sure, you should do 2 half-hour running sessions. And please read through this psycho-education: www.link-to-psycho-education.nl."
+        age = 30 # We should get this value from a database.
```
Comment (svenvanderburg) [inline Rasa_Bot/actions/actions.py:33]: Better to indicate this is a constant
```suggestion
        AGE = 30 # We should get this value from a database.
```

**Review by svenvanderburg — CHANGES_REQUESTED** [review]
Nice! 
I have 2 questions/requests:
* Did you test whether this works? (by interacting with the bot)
* You should add the paalgorithms library as a dependency 
       - add a file `requirements-actions.txt` to Rasa_Bot/actions folder (see also discussion in #43)
       - make sure the requirements are installed in the actions server dockerfile: https://github.com/PerfectFit-project/virtual-coach-server/blob/main/Rasa_Bot/Dockerfile
 * Test whether running rasa with docker works

File: Rasa_Bot/actions/actions.py (hunk 2 above)
Comment (bscheltinga) [reply Rasa_Bot/actions/actions.py:33]: @svenvanderburg thanks for the changes/feedback. I'm not familiar with these conventions. I will try my best to keep these two in mind!

I added the requirement.txt (accidently in the wrong branch at first...). I think the test with the docker file we should do together as I'm not sure how to do this.

Comment (svenvanderburg) [reply Rasa_Bot/actions/actions.py:33]: It's OK that you're not familiar yet. You'll learn along the way! :) Tip: you can use [flake8](https://flake8.pycqa.org/en/latest/) to check whether you follow the style conventions. We will also set that up in Continuous Deployment later.

Comment (svenvanderburg) [reply Rasa_Bot/actions/actions.py:33]: And let's do some pair coding! I'll propose a time slot.

**Pull request merged**